en/docs/develop/product-apis/publisher-apis/publisher-v1/publisher-v1.yaml

```diff
@@ -1512,15 +1541,19 @@ paths:
         - OAuth2Security:
           - apim:api_publish
           - apim:api_create
-      x-wso2-curl: "curl -k -H \"Authorization: Bearer f8828f52-175a-4cb2-b246-02b1e6054f97\" https://127.0.0.1:9443/api/am/publisher/v1/apis/0527a55e-8dcf-4ce5-bddf-c77e8418be8a/lifecycle"
+      x-wso2-curl: "curl -k -H \"Authorization: Bearer f8828f52-175a-4cb2-b246-02b1e6054f97\" https://127.0.0.1:9443/api/am/publisher/v1/apis/0527a55e-8dcf-4ce5-bddf-c77e8418be8a/lifecycle-state"
       x-wso2-request: |
-            GET https://127.0.0.1:9443/api/am/publisher/v1/apis/890a4f4d-09eb-4877-a323-57f6ce2ed79b/lifecycle
+            GET https://127.0.0.1:9443/api/am/publisher/v1/apis/890a4f4d-09eb-4877-a323-57f6ce2ed79b/lifecycle-state
```

Comment:
x-wso2-request, x-wso2-response, x-wso2-curl are not in use. Shall we remove them?

Reply:
fixed with https://github.com/wso2/docs-apim/pull/1812/commits/ff8d80c40e3adcb3ac46d01dcb950f0d4d164502